api/v1beta1/openstackbaremetalset_webhook.go

```diff
@@ -61,6 +63,16 @@ var _ webhook.Validator = &OpenStackBaremetalSet{}
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (r *OpenStackBaremetalSet) ValidateCreate() (admission.Warnings, error) {
 	openstackbaremetalsetlog.Info("validate create", "name", r.Name)
+	var errors field.ErrorList
```

Comment:
As I mentioned in the jira, this is redundant as we're using the nodeset name for this when creating baremetalset and there is no use-case to create these baremetalset separately.